manifests/repo/debian.pp

```diff
@@ -35,22 +31,25 @@
   Boolean $dotdeb         = true,
   Boolean $sury           = true,
 ) {
-  assert_private()
-
+  if $facts['os']['name'] != 'Debian' {
+    fail("class php::repo::debian does not work on OS ${facts['os']['name']}")
+  }
```

Comment:
Is this only applicable for Debian? What about Debian-based distros? Like ubuntu?

Reply:
In the current realization Ubuntu has it's own class - php::repo::ubuntu, so this one only for Debian.

Reply:
Ah yes, sorry for that!